templates/openstack/what-is-openstack.html

```diff
@@ -217,7 +612,130 @@ <h2>How to install OpenStack</h2>
   </div>
 </section>
 
-{% with first_item="_cloud_ubuntu_advantage", second_item="_cloud_contact_us", third_item="_cloud_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+<section class="p-strip--suru is-dark">
+  <div class="u-fixed-width">
+    <h3>Learn more about OpenStack</h3>
+    <div class="row p-divider u-no-padding--left u-no-padding--right">
```

Comment:
I don't think you need the `u-fixed-width` class and the `row` - you can just use the row. Or you can do it like you do on line 687

Reply:
@petesfrench have you fixed this? it looks the same so not sure if github is not showing the new commit or maybe you missed it (feel free to reach out if you're not sure what I mean)